### ISSUE TITLE
Feature request: Allow naming of EXE's

### ISSUE THREAD

**Issue by AllanBlackwater** [Willy-JL/F95Checker#173]
Problem: Currently, if you have a game that has multiple EXEs....different chapters, or whatever, you have to pick the EXE when you run the game, based on the path and filename.  Which isn't difficult, it's just ugly.

Suggested solution:  When adding an EXE, add an additional prompt after adding the EXE to allow you to add an option "alias".  Then, when playing the game, and it prompts for which EXE, it will use the alias if there is one, or the full path if one wasn't entered.

**Comment by Willy-JL** [comment]
i think the need for this is mitigated by the "default exe dir" option. if you have this set and keep your library organized, you would end up with a structure like
DefaultExeDir/GameName/Chapter1/GameName.exe
DefaultExeDir/GameName/Chapter2/GameName.exe

in which case, the launch prompt will just show
GameName/Chapter1/GameName.exe
GameName/Chapter2/GameName.exe
without everything else before it

**Comment by AllanBlackwater** [comment]
Fair point.  I have let some of my organizing get a bit sloppy.  I don't even remember which game was causing this frustration now.
